src/styles/blocks/button.scss

```diff
@@ -0,0 +1,24 @@
+.button {
+  min-width: 166px;
+  min-height: 40px;
+  border-radius: 5px;
+  padding: auto;
```

Comment:
The 'padding' property does not accept the value 'auto'. You should use specific values to set padding, like '0' or '10px'.